### PR TITLE
[MediaPlayer] Improve documentation

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

Improve documentation

### CODE REVIEW

File: src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void Clear()
         }
 
         /// <summary>
-        /// Gets the number of items.
+        /// Gets the number of equalizer bands available
```
Comment (neoiomok) [inline src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs:101]: please add period

File: src/Tizen.Multimedia.MediaPlayer/Player/MediaBufferSource.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public sealed class MediaBufferSource : MediaSource
         private byte[] _buffer;
 
         /// <summary>
-        /// Initializes a new instance of the MediaBufferSource class with an allocated buffer.
+        /// Initializes a new instance of the MediaBufferSource class with the specified buffer langth.
```
Comment (neoiomok) [inline src/Tizen.Multimedia.MediaPlayer/Player/MediaBufferSource.cs:36]: typo error
langth -> length

File: src/Tizen.Multimedia.MediaPlayer/Player/EqualizerBand.cs [hunk 3]
```diff
@@ -85,7 +85,7 @@ public int Level
 
 
         /// <summary>
-        /// Gets the frequency in dB.
+        /// Gets the frequency in dB of the equalizer band..
```
[the remark this answers is not in the document]
Comment (hsgwon) [reply src/Tizen.Multimedia.MediaPlayer/Player/EqualizerBand.cs:88]: I'll fix it.

File: src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.MediaPlayer/Player/AudioEffect.cs:101]: I'll fix it.

File: src/Tizen.Multimedia.MediaPlayer/Player/MediaBufferSource.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.MediaPlayer/Player/MediaBufferSource.cs:36]: I'll fix it.

**Pull request merged**